app/config/app.config.js

```diff
@@ -0,0 +1,294 @@
+import OlStyleStyle from 'ol/style/Style';
+import OlStyleFill from 'ol/style/Fill';
+import OlStyleCircle from 'ol/style/Circle';
+import OlStyleStroke from 'ol/style/Stroke';
+
+class AppConfig {
+
+  /**
+   * The primary color.
+   * @type {String} primaryColor
+   */
+  primaryColor = '#0e1058';
+
+  /**
+   * The secondary color.
+   * @type {String} secondaryColor
+   */
+  secondaryColor = '#b94800';
+
+  /**
+   * Configure the application header here.
+   * @type {Object} header
+   */
+  header = {
+    /**
+     * The path to the logo image used for the application. Relative to the
+     * resources img folder.
+     * @type {String} logo
+     */
+    logo: 'BMEL_Logo.svg',
```

Comment:
Change this to the terrestris logo and use the config in the corresponding position.